src/argilla/datasets/__init__.py

```diff
@@ -38,3 +63,19 @@ def configure_dataset(name: str, settings: Settings, workspace: Optional[str] =
     active_api = api.active_api()
     datasets = active_api.datasets
     datasets.configure(name, workspace=workspace or active_api.get_workspace(), settings=settings)
+
+
+def configure_dataset(name: str, settings: Settings, workspace: Optional[str] = None) -> None:
+    """
+    Configures a dataset with a set of configured labels. If dataset does not
+    exist yet, an empty dataset will be created.
+
+    A subset of settings can be provided.
+
+    Args:
+        name: The dataset name
+        settings: The dataset settings
+        workspace: The workspace name where the dataset will belongs to
+    """
+    _LOGGER.warning("This method is deprecated. Use configure_dataset_settings instead.")
```

Comment:
Does it make sense to set `logging.captureWarnings(capture=True)` to log this warning as a `DeprecationWarning` using `warnings`? More information at https://docs.python.org/3/library/logging.html#logging.captureWarnings

Reply:
Ok, I'll add this comment there then!